base_layer/core/src/base_node/proto/request.proto

```diff
@@ -14,18 +14,20 @@ message BaseNodeServiceRequest {
         HashOutputs fetch_kernels = 3;
         // Indicates a FetchHeaders request.
         BlockHeights fetch_headers = 4;
+        // Indicates a FetchHeadersWithHashes request.
+        HashOutputs fetch_headers_with_hashes = 5;
```

Comment:
Just FYI, we'll need to freeze message numbers as soon as testnet is live; from that point additional messages will have to be appended; to help with message compatibility across different node versions